test_data/rules/unnecessary_late.dart

```diff
@@ -8,6 +8,10 @@ late String unnecessaryTopLevelLate = ''; // LINT
 
 late String necessaryTopLevelLate; // OK
 
+late String unnecessaryListLateOne = '', unnecessaryListLateTwo = ''; // LINT
```

Comment:
If we go with the second purpose, it would be good to put each declaration on a separate line to make it clear that we're getting two lints here.